packages/cli/README.md

```diff
@@ -50,94 +50,18 @@ Dump the screenshots with config file
 
 ```
 
-## Usage -- sprites
+## Usage -- Create Config
 
-Generate a sprite sheet from a list of sprites
-
-```
-./bin/bmc.js sprites --ratio 1 --ratio 2 --path ./config/sprites/topographic --output assets
-```
-
-Outputs:
-
-assets/topographic.json
-assets/topographic.png
-
-assets/topographic@2x.json
-assets/topographic@2x.png
-
-## Usage -- cogify
-
-Create a collection of cloud optimized geotiff's from a collection of geotiff, that is optimized to be used in `@basemaps/tiler`
-
-Create a list of COG's to create
-
-```bash
-./bin/bmc.js -V job --source ./source_folder/ --output ./source_folder/cogify/
-```
-
-Build a specific COG
+Create a individual config.json from a path of imageries for Basemaps to load as standalone server to view them. This is mainly used for the preview images for the Basemaps imagery import process.
 
 ```bash
-./bin/bmc.js -V cog --job ./cogs/01DYREBEEFFXEPBAYBED2TMAFJ/job.json --name 1-2-3 --commit
+./bin/bmc.js create-config --path s3://..../images/*.tiff --title image_title --commit
 ```
 
-Make cog job json
-
-```bash
-./bin/bmc.js -V make-cog --imagery s3://imagery_path --target destination_bucket --name imagery_name --tile-matrix NZTM2000Quad --output ./jobs.json
-```
+## Usage -- Overview
 
-Split make cog job into chunked sub-jobs
+Create a overview tar file of tiles from the given of imageries. This create low zoom level cogs for the individual imagery and significantly improve the performance of serving individual imagery by pre-render and save the cogs in overview instead of render them on the fly.
```

Comment:
```suggestion
Create a overview tar file of tiles from the given of imagery. This create low zoom level tiles for the individual imagery and significantly improve the performance of serving individual imagery by pre-rendering and saving the tiles instead of rendering them on the fly.
```